app/components/folder/draggable-dropzone.js

```diff
@@ -0,0 +1,25 @@
+import Component from '@ember/component';
```

Comment:
Add js docs with this info:
``
/** 
* @see https://github.com/passwordcockpit/frontend for the canonical source repository 
* @copyright Copyright (c) 2018 Blackpoints AG (https://www.blackpoints.ch) 
* @license https://github.com/passwordcockpit/frontend/blob/master/LICENSE.md BSD 3-Clause License 
*/

``

Reply:
Use the jsdocs above to edit all files with the same problem